pytket/extensions/qiskit/backends/aer.py

```diff
@@ -275,14 +282,15 @@ def process_circuits(
             if self._needs_transpile:
                 qcs = transpile(qcs, self._qiskit_backend)
 
-            seed = cast(Optional[int], kwargs.get("seed"))
             job = self._qiskit_backend.run(
                 qcs,
                 shots=n_shots,
                 memory=self._memory,
                 seed_simulator=seed,
                 noise_model=self._noise_model,
             )
+            if kwargs.get("seed_auto_increase") and type(seed) is int:
+                seed += 1
```

Comment:
```suggestion
            seed += 1
```

Reply:
Done in [85489ef](https://github.com/CQCL/pytket-qiskit/pull/197/commits/85489ef983300e4739976ba68048e55e5ceb932a)